### PR TITLE
Add materialize to the ntsb loader for luna

### PR DESCRIPTION
**Pull request by eric-anderson**
Also add a simple query script, and fix some version consistency issues.

### CODE REVIEW

File: examples/luna/ntsb_loader.py [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+
+
+context = sycamore.init()
+docset = context.read.binary(SOURCE_DATA_PATH, binary_format="pdf")
+tokenizer = HuggingFaceTokenizer("thenlper/gte-small")
+llm = OpenAI(OpenAIModels.GPT_3_5_TURBO.value)
+
+# partitioning docset
+partitioned_docset = (
+    docset.partition(partitioner=UnstructuredPdfPartitioner())
```
Comment (mdwelsh) [inline examples/luna/ntsb_loader.py:122]: Can we use ArynPartitioner please

File: examples/luna/ntsb_loader.py [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+    # these are here mostly as examples; the last materialize will at this point take
+    # effect, you can use these during testing.
+    .materialize(path="/tmp/after_partition", source_mode=sycamore.MaterializeSourceMode.IF_PRESENT)
+    .map(add_property_to_schema)
+    .extract_properties(property_extractor=OpenAIPropertyExtractor(llm=llm, num_of_elements=35))
+    .materialize(path="/tmp/after_llm", source_mode=sycamore.MaterializeSourceMode.IF_PRESENT)
```
Comment (mdwelsh) [inline examples/luna/ntsb_loader.py:128]: Why are we doing this here - just as an example? Seems like this should be an option or the directories we're materializing to should be using `tempdir` (or at least named constants at the top of this file).

File: examples/luna/ntsb_loader.py [hunk 3]
```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env python
```
Comment (mdwelsh) [inline examples/luna/ntsb_loader.py:1]: We're not using the name "luna" anywhere in the code - this is Sycamore Query, so I'd recommend calling this directory `examples/query`.

File: examples/luna/ntsb_loader.py [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+    .sketch()
+    .embed(embedder=SentenceTransformerEmbedder(batch_size=100, model_name="sentence-transformers/all-MiniLM-L6-v2"))
+    # comment out, to force re-evaluation of the pipeline, otherwise since this always exists it
+    # will be used.
+    .materialize(
+        path="s3://aryn-public/materialize/examples/luna/ntsb_loader",
+        source_mode=sycamore.MaterializeSourceMode.IF_PRESENT,
+    )
+    # materialize locally after reading from S3, it's a bit faster if you're running rmeotely
+    .materialize(path="/tmp/after_embed", source_mode=sycamore.MaterializeSourceMode.IF_PRESENT)
```
Comment (mdwelsh) [inline examples/luna/ntsb_loader.py:142]: Same here

File: pyproject.toml [hunk 5]
```diff
@@ -36,6 +37,7 @@ remote-processors = { path = "lib/remote-processors", develop = true }
 integration = { path = "apps/integration", develop = true }
 sycamore-poetry-lock = { path = "lib/poetry-lock", develop = true }
 aryn-sdk = { path = "lib/aryn-sdk", develop = true }
+queryui = { path = "apps/query-ui", develop = true }
```
Comment (mdwelsh) [inline pyproject.toml:40]: I don't think this dependency needs to be here, right? queryui depends on sycamore, not the other way around.

Comment (eric-anderson) [reply pyproject.toml:40]: If it's not there, when you do a poetry install in the root you don't get the stuff to run the query ui.

File: examples/luna/ntsb_loader.py (hunk 1 above)
Comment (eric-anderson) [reply examples/luna/ntsb_loader.py:122]: Done, but scares me a lot that as soon as I changed the partitioner I got different results.

File: examples/luna/ntsb_loader.py (hunk 2 above)
Comment (eric-anderson) [reply examples/luna/ntsb_loader.py:128]: They're there as examples and to help while I was getting it working.

File: examples/luna/ntsb_loader.py (hunk 3 above)
Comment (eric-anderson) [reply examples/luna/ntsb_loader.py:1]: Done.

File: examples/luna/ntsb_loader.py (hunk 4 above)
Comment (eric-anderson) [reply examples/luna/ntsb_loader.py:142]: Also changed.

File: examples/luna/ntsb_loader.py (hunk 3 above)
Comment (mdwelsh) [reply examples/luna/ntsb_loader.py:1]: Did you push your changes? This file is still called `examples/luna/ntsb_loader.py`

File: examples/luna/ntsb_loader.py (hunk 4 above)
Comment (mdwelsh) [reply examples/luna/ntsb_loader.py:142]: I don't see changes here

**Review by mdwelsh — APPROVED** [review]

**Pull request merged**